src/isar/apis/schedule/start_echo_mission.py

```diff
@@ -8,7 +8,9 @@
 from isar.config import config
 from isar.models.communication.messages import StartMessage, StartMissionMessages
 from isar.models.mission import Mission
-from isar.services.service_connections.echo.echo_service import EchoServiceInterface
+from isar.services.service_connections.echo.echo_service import (
+    EchoServiceInterface,
+)
```

Comment:
Double check line length. Standard for our project has been 88, so we should be consistent.